### ISSUE TITLE
Add start/stop autoplay actions to amp-carousel

### ISSUE THREAD

**Issue by cvializ** [ampproject/amphtml#11173]
An AMP implementer needs start/stop on `amp-carousel` for ADA compliance.

[From a Slack channel question](https://amphtml.slack.com/archives/C0ADR0A0K/p1504628786000350)

/cc @aghassemi

**Comment by nainar** [comment]
Hi, I am claiming this issue.
